aws-resource-exporter-config.yaml

```diff
@@ -0,0 +1,9 @@
+default:
+  regions: "us-east-1"
+  timeout: 10s
+rds: {}
```

Comment:
i like the open structure of the config file overall.
not asking you to do anything about it, but fwiw, this is another section we want to move to this config file eventually: https://github.com/app-sre/aws-resource-exporter/blob/ffe2c710b3a472c53c3d437c4d10037ce789352e/rds.go#L45-L209

Reply:
Yes the current ConfigFile just uses `DefaultConfig` structs for all sections, but I was thinking of just replacing any of those with a struct that can include the `DefaultConfig` and add additional fields for specific configuration.